pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -254,6 +275,16 @@ func getCertsFromController(ctx *zedagentContext) bool {
 		return false
 	}
 
+	// Did the certificate change?
+	_, err = os.Stat(types.ServerSigningCertFileName)
+	if err == nil {
+		oldCertBytes, err := ioutil.ReadFile(types.ServerSigningCertFileName)
```

Comment:
ioutil is deprecated using `os.ReadFile`

Reply:
@shjala yes I think the fix for that just made it to master.